### ISSUE TITLE
Bug in page:Download and install 

### ISSUE THREAD

**Issue by craigsapp** [kripken/emscripten#4074]
In OS X, the command-line call:

```bash
 # Fetch the latest registry of available tools.
./emsdk update
```
returns this error:

```
$ ./emsdk update
Traceback (most recent call last):
  File "./emsdk", line 1797, in <module>
    sys.exit(main())
  File "./emsdk", line 1709, in main
    update_emsdk()
  File "./emsdk", line 1257, in update_emsdk
    download_and_unzip(urljoin(emsdk_packages_url, 'emsdk_unix_update.tar.gz'), emsdk_path(), download_even_if_exists=True)
  File "./emsdk", line 687, in download_and_unzip
    dst_file = download_file(urljoin(emsdk_packages_url, zipfile), 'zips/', download_even_if_exists, filename_prefix)
  File "./emsdk", line 352, in download_file
    u = urlopen(url)
  File "/usr/local/Cellar/python/2.7.10_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 154, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/local/Cellar/python/2.7.10_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 431, in open
    response = self._open(req, data)
  File "/usr/local/Cellar/python/2.7.10_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 449, in _open
    '_open', req)
  File "/usr/local/Cellar/python/2.7.10_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 409, in _call_chain
    result = func(*args)
  File "/usr/local/Cellar/python/2.7.10_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 1240, in https_open
    context=self._context)
  File "/usr/local/Cellar/python/2.7.10_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 1197, in do_open
    raise URLError(err)
urllib2.URLError: <urlopen error [Errno 64] Host is down>
```
It seems that the URL base trying to be accessed is:

```python
  emsdk_master_server = 'https://s3.amazonaws.com/mozilla-games/emscripten/'
```

Trying to directly access the URL:
```
https://s3.amazonaws.com/mozilla-games/emscripten/emsdk_unix_update.tar.gz
```
Gives the error:

```
This XML file does not appear to have any style information associated with it. The document tree is shown below.
<Error>
<Code>AccessDenied</Code>
<Message>Access Denied</Message>
<RequestId>CD72A580FD7F63D5</RequestId>
<HostId>
Yi4d/uncPC6WwVmIIH+oMPzp+c3VTHj9cSb08dbD6DPCz05sPgkLkUXTTEY9Ry4Z
</HostId>
</Error>
```

URL: ../../docs/getting_started/downloads

**Comment by juj** [comment]
Hey, was this temporary or are you still seeing this? Tried locally on OS X with `./emsdk update`, which gives the following for me

	Jukkas-MacBook-Pro:emsdk_git jjylanki$ ./emsdk update --verbose
	Downloading: /Users/jjylanki/emsdk_git/zips/emsdk_unix_update.tar.gz from https://s3.amazonaws.com/mozilla-games/emscripten/packages/emsdk_unix_update.tar.gz
	Unpacking '/Users/jjylanki/emsdk_git/zips/emsdk_unix_update.tar.gz' to '/Users/jjylanki/emsdk_git'
	x emscripten-tags.txt
	x emsdk
	x emsdk_env.sh
	x emsdk_manifest.json
	x README.md
	Fetching all tags from Emscripten Github repository...
	Done. 76 tagged releases available, latest is 1.36.4.

Can you try again, if the networking issue was transient?

**Comment by craigsapp** [comment]
I just tried it, and it is working.
